### ISSUE TITLE
When I specify language tag to be 'en' to customise just a few string, I see missing keys warnings in the console

### ISSUE THREAD

**Issue by badikumar** [onfido/onfido-sdk-ui#532]
### What was the expected behaviour?

When I specify language tag to be 'en' in order to customise just a few string for the english phrases, I see missing keys warnings in the console. I shouldn't see the warning messages. it should just use the default values for the strings that weren't supplied.

### What happened instead?

Instead, I see warning message giving a long list of keys that are missing.

### Version info:

- Onfido SDK (Release):

### Integration configuration:

      language: {
        locale: 'en',
        phrases: {
         'capture.driving_licence.front.instructions':
            'Upload front of license from your computer. Ensure it is a colour copy of the original.',
          national_identity_card: 'Identiy Card or Book',
       }
     }

### Steps to reproduce:

customize a few strings using the language key. and load onfido SDK on the browser. The console has lots of warnings about missing keys.

**Comment by jamesmosier** [comment]
Is it possible to import the other locale strings so they can be merged with my custom content? Does the SDK export those locales? @rfreitas 

Because there is no way to override copy without a ton of warnings, I was hoping to do something like:

```
import { init, locales } from 'onfido-sdk-ui';

init({
  token: 1234,
  containerId: 'my-element',
  onComplete: () => {},
  language: {
    language: 'en',
    locale: 'us',
    phrases: {
      welcome: { 
        next_button: 'Next ->', 
        ...locales.en.welcome 
      },
      ...locales.en
    }
  }
})
```

**Comment by stefaniacardenas** [comment]
Unfortunately, that's not possible at the moment but we are going to disable the warnings in the release after the next. I will post an update here when it's live.

**Comment by jamesmosier** [comment]
Thanks @stefaniacardenas! I'd be happy to open a PR with a new option to disable those warnings, unless you all already have something in mind for that fix.

**Comment by stefaniacardenas** [comment]
The plan was to disable the "missing keys" warning for supported languages (i.e. `en` and `es`) if only some of the strings are provided. Thanks @jamesmosier, feel free to raise a PR. That's very much appreciated.

**Comment by stefaniacardenas** [comment]
Hi, this has been fixed in version [5.4.0](https://github.com/onfido/onfido-sdk-ui/releases/tag/5.4.0). Thank you very much @jamesmosier for the PR. I will now close this issue.
